plugins/woocommerce/src/Admin/DeprecatedClassFacade.php

```diff
@@ -58,7 +58,9 @@ class DeprecatedClassFacade {
 	 * Constructor.
 	 */
 	public function __construct() {
-		$this->instance = new static::$facade_over_classname();
+		if ( '' !== static::$facade_over_classname ) {
+			$this->instance = new static::$facade_over_classname();
```

Comment:
The `FeatureGating` class doesn't have a replacement class, so I updated this to allow for that.

This way it still catches the class initialization and any function calls and only shows the deprecated message.